includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -41,11 +41,12 @@ public function __construct() {
 	 * @since 1.0.0
 	 * @version 4.0.0
 	 */
-	public function add_admin_notice( $slug, $class, $message, $dismissible = false ) {
+	public function add_admin_notice( $slug, $class, $message, $dismissible = false, $icon_img_url = '' ) {
```

Comment:
Is it worth making this argument type explicit?
```suggestion
	public function add_admin_notice( $slug, $class, $message, $dismissible = false, string $icon_img_url = '' ) {
```

Reply:
I have removed the icon following the latest suggestion from Orcun above. So, this parameter is no longer needed.